examples/grid3d_crop.py

```diff
@@ -49,12 +48,12 @@ def cropper():
             tmpgrd = grd.copy()
             tmpgrd.crop(ncr, nrr, (1, nlay), props="all")
             # save to disk as ROFF files
-            tmpgrd.to_file("grid" + fname + ".roff")
+            tmpgrd.to_file("/tmp/grid" + fname + ".roff")
```

Comment:
Drop `/` in string for cross platform compatibility?

Reply:
Agree; I make it all using tempfile and pathlib functions